Quantum-Control-Applications/Superconducting/Single-Fixed-Transmon/04b_resonator_spectroscopy_wide_range_octave.py

```diff
@@ -0,0 +1,175 @@
+"""
+        RESONATOR SPECTROSCOPY OVER A WIDE RANGE with the Octave
+This sequence involves measuring the resonator by sending a readout pulse and demodulating the signals to extract the
+'I' and 'Q' quadratures across varying readout intermediate and LO frequencies. The Octave LO frequency is swept in the
+outer loop to optimize run time.
+The Octave port can be calibrated at all the involved LO frequencies and at the median intermediate frequency.
+The correction parameters are updated in Python after setting the LO frequency.
+The data is then post-processed to determine the resonator resonance frequency.
+This frequency can be used to update the readout intermediate frequency in the configuration under "resonator_IF".
+
+Prerequisites:
+    - Ensure calibration of the time of flight, offsets, and gains (referenced as "time_of_flight").
+    - Calibrate the IQ mixer connected to the readout line (whether it's an external mixer or an Octave port).
```

Comment:
This experiment is not possible with an external mixer as the ´calibrate_element´ method acts on the Octave mixer. This comment might be a bit misleading. 
```suggestion
    - Calibrate the Octave port connected to the readout line.
```